src/components/Rating/__test__/ratingItems.spec.js

```diff
@@ -10,10 +10,30 @@ describe('<RatingItems />', () => {
             name: 'name-1',
             onChange: expect.any(Function),
             filled: true,
+            half: false,
+            readOnly: false,
         });
     });
     it('should render the right amount of Star components', () => {
         const component = mount(<RatingItems />);
         expect(component.children().length).toBe(5);
     });
+    it('should render half star at the right position when value is a float', () => {
+        const component = mount(<RatingItems value={'3.5'} />);
+        const star = component.childAt(3);
+        expect(star.find('StarHalf').exists()).toBe(true);
+    });
```

Comment:
I think this test belongs to ```src/components/Rating/star.js``` component since it is who decides what icon render
the test should check that the StarHalf icon is rendered when the half prop is passed